src/shogun2-core/shogun2-model/src/main/java/de/terrestris/shogun2/model/module/Map.java

```diff
@@ -0,0 +1,169 @@
+/**
+ *
+ */
+package de.terrestris.shogun2.model.module;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.persistence.CascadeType;
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.OneToOne;
+import javax.persistence.OrderColumn;
+import javax.persistence.Table;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+import de.terrestris.shogun2.model.layer.Layer;
+import de.terrestris.shogun2.model.map.MapConfig;
+
+/**
+ *
+ * Class represents a
+ * <a href="https://github.com/geoext/geoext3/blob/master/src/component/Map.js">map component</a>
+ *
+ * @author Andre Henn
+ * @author Daniel Koch
+ * @author terrestris GmbH & Co. KG
+ *
+ */
+@Entity
+@Table
+public class Map extends Module {
+
+	/**
+	 *
+	 */
+	private static final long serialVersionUID = 1L;
+
+	/**
+	 * The MapConfig used by this Map. A MapConfig can be used by several maps
+	 * or overview maps.
+	 */
+	@OneToOne(
```

Comment:
According to the JavaDoc above:

*A MapConfig can be used by several maps or overview maps.*,

shouldn't this be a `@ManyToOne` relation?